### PR TITLE
SAK-42835 Feedback: Update banners to use the Sakai ones.

### PR DESCRIPTION
**Pull request by mpellicer**

### CODE REVIEW

File: feedback/src/webapp/WEB-INF/bootstrap.jsp [hunk 1]
```diff
@@ -78,15 +78,11 @@
         <div class="portletBody">
             <ul id="feedback-toolbar" class="navIntraTool actionToolBar" role="menu"></ul>
             <div class="portletBody">
-                <div id="feedback-error-message-wrapper">
-                    <div>
-                        <span></span><a href="javascript:;" alt="${i18n.close}" title="${i18n.close}"></a>
-                    </div>
+                <div id="feedback-error-message-wrapper" class="sak-banner-error">
+                    <span></span>
```
Comment (adrianfish) [inline feedback/src/webapp/WEB-INF/bootstrap.jsp]: should the link be taken out?

Comment (fostersdesign) [reply feedback/src/webapp/WEB-INF/bootstrap.jsp]: We don't have a working dismissible banner component defined yet, although it is on the list for SWITCH. 

For now, it matches other tools to have the message stay on the page until the page is refreshed.

Comment (mpellicer) [reply feedback/src/webapp/WEB-INF/bootstrap.jsp]: Exactly, I think the link should be added globally, not at a tool level.

File: feedback/src/webapp/WEB-INF/bootstrap.jsp [hunk 2]
```diff
@@ -78,15 +78,11 @@
...
                 </div>
-                <div id="feedback-info-message-wrapper">
-                    <div>
-                        <span></span><a href="javascript:;" alt="${i18n.close}" title="${i18n.close}"></a>
-                    </div>
+                <div id="feedback-info-message-wrapper" class="sak-banner-success">
+                    <span></span>
```
[the remark this answers is not in the document]
Comment (mpellicer) [reply feedback/src/webapp/WEB-INF/bootstrap.jsp]: Yes it is!

**Pull request merged**